SteamLicenseRemover.js

```diff
@@ -1,42 +1,39 @@
 // ==UserScript==
     // @name         Steam License Remover
     // @namespace
-    // @version      1.1
-    // @description  Remove any "Free" games from your Steam Library by removing the game's license from your account.  
+    // @version      1.2
+    // @description  Remove any "Free" games from your Steam Library by removing the game's license from your account.
     // @author       IroN404
     // @match        https://store.steampowered.com/account/licenses/
 
 
-const table = document.querySelector('.account_table');
-const rows = table.querySelectorAll('tr');
-const total = rows.length;
+const removeLinkEls = document.querySelectorAll('.free_license_remove_link > a');
+let itemIds = [];
+
+removeLinkEls.forEach(el => itemIds.push(el.getAttribute('href').match(/\d+/)[0]));
+
+const total = itemIds.length;
+console.log(`Starting removal of ${total} entries`);
+const start = Date.now();
 
 let removed = 0;
-const intervalId = setInterval(() => {
-    rows.forEach(row => {
-        const div = row.querySelector('.free_license_remove_link');
-        if(div) {
-            const removeLink = div.querySelector('a');
-            if (removeLink) {
-                removed += 1;
-                const link = removeLink.getAttribute('href');
-                const id = link.split('(')[1].split(',')[0].trim();
-                jQuery.post(
-                    'https://store.steampowered.com/account/removelicense',
-                    {
-                        sessionid: g_sessionID,
-                        packageid: id
-                    }
-                );
-            }
-        }
+const intervalId = setInterval(async () => {
```

Comment:
This can still cause issues if the fetch calls take longer than 2 seconds to complete, use setTimeout after the fetch completes instead.